### ISSUE TITLE
Javacpp - tesseract : TessBaseApi, Popup Visual c++ error on exit

### ISSUE THREAD

**Issue by gr3gdev** [bytedeco/javacpp-presets#142]
I am trying to use Tesseract to have OCR functionality in a Java application.
When the Java program finishes, the JVM crashes.

##### An code example :#####

```java
public class Example {

    public static void main(final String[] args) {
        final TessBaseAPI api = new TessBaseAPI();
    }

}
```

It gives the following error :

**Comment by saudet** [comment]
Which version? What OS?

**Comment by saudet** [comment]
It might be the same problem as this: http://stackoverflow.com/q/34122497/523744

In which case I recommend to try to build with the latest version of MSYS2, which appears to contain a fix for this bug.

**Comment by gr3gdev** [comment]
OS : Windows

This is a maven project :
```xml
<dependency>
	<groupId>org.bytedeco.javacpp-presets</groupId>
	<artifactId>tesseract</artifactId>
	<version>3.04-1.1</version>
</dependency>
<dependency>
	<groupId>org.bytedeco.javacpp-presets</groupId>
	<artifactId>tesseract</artifactId>
	<version>3.04-1.1</version>
	<classifier>windows-x86</classifier>
</dependency>
```

**Comment by saudet** [comment]
Right, please try to rebuild with the latest version of MSYS2 and let me know if that changes anything! Thanks

**Comment by saudet** [comment]
This should be fixed with the release of version 1.2, but let me know if you still see that happening. Thanks for reporting!
